README.md

```diff
@@ -7,4 +7,59 @@ Course in Ecology and Evolutionary Biology
 [![Build Status](https://travis-ci.org/UofTCoders/rcourse.svg?branch=master)](https://travis-ci.org/UofTCoders/rcourse)
 [![DOI](https://zenodo.org/badge/97400494.svg)](https://zenodo.org/badge/latestdoi/97400494)
 
+
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying 
```

Comment:
Change first sentence to: "Statistical analysis, modelling, and data simulation are essential skills for ecologists and evolutionary biologists."